### ISSUE TITLE
Error: less 

### ISSUE THREAD

**Issue by Baron2011** [nocobase/nocobase#2131]
## Desktop (please complete the following information)

- Browser [e.g. chrome v102, safari]

**Comment by dream2023** [comment]
Did you install it with  `yarn` or other command？

**Comment by Baron2011** [comment]
yes, I did. Installation is finished via yarn command refer to the online docs ( https://docs.nocobase.com/welcome/getting-started/upgrading/create-nocobase-app #Minor version upgrade). I found this issue from 0.10.0-alpha.2 and reinstalled it the issue is still the same when run "yarn build" or "yarn nocobase upgrade" .

**Comment by Baron2011** [comment]
More info: I installed NocoBase App refer to the online docs ( https://docs.nocobase.com/welcome/getting-started/installation/create-nocobase-app) , and then I found some issues and tried to upgrade via Minor version Upgrade, then the issue occured as described above.

**Comment by dream2023** [comment]
I cannot reproduce this error, but if you want to resolve it, you can follow these steps:

1. modify `package.json`
![image](https://github.com/nocobase/nocobase/assets/19297757/0ed15318-13d9-4a37-acbd-4c15b416d2e6)

2. `yarn install`
